src/ducks/upload/UploadQueue.jsx

```diff
@@ -38,6 +38,33 @@ const Item = translate()(({ t, file, status }) => {
   )
 })
 
+const Header = translate()(({ t, total, done, successful, onDoubleClick, onClose }) => (
+  <h4 className={styles['upload-queue-header']} onDoubleClick={onDoubleClick}>
+    {
+      done < total &&
+      <div className={styles['upload-queue-header-inner']}>
+        <span className='coz-desktop'>
+          {t('UploadQueue.header', { smart_count: total })}
+        </span>
+        <span className='coz-mobile'>
+          {t('UploadQueue.header_mobile', { done, total })}
+        </span>
+      </div>
+    }
+    {
+      done >= total &&
+      <div className={styles['upload-queue-header-inner']}>
+        <span>
+          {t('UploadQueue.header_done', { done: successful, total })}
+        </span>
+        <button className={classNames(styles['btn-close'])} onClick={onClose}>
+          { t('UploadQueue.close') }
+        </button>
+      </div>
+    }
+  </h4>
+))
```

Comment:
If `done` exceeds the threashold, we render a `<Close />` component (composed by a `<span />` and a `<button />`.
If not, we render a `<OnGoing />` component (composed by two `<span />` elements for desktop and mobile).

Both `<Close />` and `<OnGoing />` components are wrapped in the same div.

Looking the code, it appears less clear than my above explanation. Just a matter of abstraction and readability. Nothing's bad, nothing's blocking, nothing's awkward, but improvable.